server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/CassandraMailQueueBrowser.java

```diff
@@ -111,7 +110,7 @@ public boolean hasNext() {
         return browseStartDao.findBrowseStart(queueName)
             .flatMapMany(this::allSlicesStartingAt)
             .flatMapSequential(slice -> browseSlice(queueName, slice))
-            .subscribeOn(Schedulers.parallel());
+            .subscribeOn(Schedulers.elastic());
```

Comment:
`boundedElastic`?

Reply:
We had a thread starvation issue with `boundedElastic` if I remember correctly, so it has been decided to go back to `elastic`